### PR TITLE
Change of GetVelocity API parameter from projectId to projectSlug

### PR DESCRIPTION
**Pull request by psing161**
ProjectSlug accepting as a input from UI

### CODE REVIEW

**Review by AshankDsouza — CHANGES_REQUESTED** [review]
Please could we give meaningful names to PRs?
It becomes very hard to debug without meaningful names

File: src/main/java/com/kansas/TaigaAPI/controller/MilestoneController.java [hunk 1]
```diff
@@ -135,4 +136,9 @@ public JsonNode getGivenSprintData(int sprintId, JsonNode allSprintData){
         return null;
     }
 
+    @GetMapping("/{projectId}/getTotalPoints")
```
Comment (AshankDsouza) [inline src/main/java/com/kansas/TaigaAPI/controller/MilestoneController.java]: Should it not be projectSlug?

File: src/main/java/com/kansas/TaigaAPI/service/TasksService.java [hunk 2]
```diff
@@ -173,4 +173,6 @@ else if (taskData.get("code").asText().equals("token_not_valid")){
 
     }
 
+
+
```
Comment (AshankDsouza) [inline src/main/java/com/kansas/TaigaAPI/service/TasksService.java]: lets avoid these non-meaningful changes to decrease number of files changed.

**Review by AshankDsouza — CHANGES_REQUESTED** [review]

File: src/main/java/com/kansas/TaigaAPI/controller/MilestoneController.java (hunk 1 above)
Comment (AshankDsouza) [reply src/main/java/com/kansas/TaigaAPI/controller/MilestoneController.java]: I sent the url format the frontend is expecting, please check on slack

Comment (psing161) [reply src/main/java/com/kansas/TaigaAPI/controller/MilestoneController.java]: addressed

**Review by AshankDsouza — APPROVED** [review]
Looks good to me.

**Pull request merged**